junebug/command_line.py

```diff
@@ -61,6 +61,14 @@ def parse_arguments(args):
     parser.add_argument(
         '--amqp-password', '-amqppass', dest='amqp_pass', type=str,
         help='The password to use for the amqp auth. Defaults to "guest"')
+    parser.add_argument(
+        '--message-ttl', '-ttl', dest='ttl', type=int,
+        help='The maximum time allowed to reply to a message. Defaults to '
+        '"60" seconds')
```

Comment:
I would vote for `inbound-message-ttl` and `outbound-message-ttl` being separate options (given our experience with SMPP ttls).